dts/bindings/iio/adc/nuvoton,npcx-adc.yaml

```diff
@@ -18,6 +18,14 @@ properties:
         type: phandles
         required: true
         description: configurations of pinmux controllers
+    threshold-count:
```

Comment:
Besides the unused `#define`, does this get used?

Reply:
This is for PR #40904, which adds threshold functionality.

Reply:
Remove it from this PR then, and add it to the appropriate PR.

Reply:
done.